.changeset/breezy-schools-compete.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Fixes #10348
```

Comment:
This isn't very helpful in a lot of places? 😅 Perhaps better to just describe what the issue was instead

Reply:
Oh wut I never committed the updated text, will fix